pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BaseBrokerStarter.java

```diff
@@ -146,6 +147,7 @@ public abstract class BaseBrokerStarter implements ServiceStartable {
   protected HelixExternalViewBasedQueryQuotaManager _queryQuotaManager;
   protected MultiStageQueryThrottler _multiStageQueryThrottler;
   protected AbstractResponseStore _responseStore;
+  protected BrokerGrpcServer _brokerGrpcServer;
```

Comment:
Do we want this to be always enabled or configurable?

Reply:
I would love to have it enable by default with the same TLS as jetty

Reply:
TLS port is not enabled by default, and requires explicit configuration. My concern is that certain users might not want to assign a separate port for gRPC, but rather not opening it. I'd suggest checking the configured port, and only start the gRPC server when the port is positive (same as how TLS port is used)

Reply:
default not start the BrokerGrpcServer